src/test/java/org/apache/bcel/verifier/VerifierMainTestCase.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.bcel.verifier;
+
+import org.apache.bcel.AbstractTestCase;
+import org.apache.commons.lang3.StringUtils;
+import org.eclipse.jdt.internal.compiler.batch.Main;
+import org.junit.jupiter.api.Test;
+
+import java.io.BufferedInputStream;
+import java.io.PrintWriter;
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class VerifierMainTestCase extends AbstractTestCase {
+
+    @Test
+    public void testSWAP() throws Exception {
+        final String[] argv = new String[] { "src/test/java/org/apache/bcel/data/SWAP.java", "-g", "-source", "1.4", "-target", "1.4", "-d", "target/test-classes" };
+        new Main(new PrintWriter(System.out), new PrintWriter(System.err), false/*systemExit*/, null/*options*/, null/*progress*/).compile(argv);
```

Comment:
This line answers that requirement "To be compiled with Eclipse Compiler (ecj) with source/target 1.4."

Reply:
I'll update the test so that it checks for swap and update the comment.

Reply:
👍